client/galaxy/scripts/utils/uploadbox.js

```diff
@@ -97,7 +97,7 @@
         }, false);
 
         // send request
-        Galaxy.emit.debug('uploadbox::uploadpost()', 'Posting following data.', cnf);
+        Galaxy.logger.emit('debug', 'uploadbox::uploadpost()', [ 'Posting following data.', cnf ]);
```

Comment:
Well - the whole point was that people wouldn't have to add a namespace on **every** call. I'd rather not have two ways to call the same method because that creates sprawl and becomes hard to change (like perl), but I think you and I use namespaces differently.

If you prefer this way and don't want to modularize uploadbox: ok - I don't particularly want to fight about it.